app/src/main/AndroidManifest.xml

```diff
@@ -89,13 +89,13 @@
                 <category android:name="android.intent.category.DEFAULT" />
                 <category android:name="android.intent.category.BROWSABLE" />
                 <data android:scheme="com.amaze.filemanager" />
-                <data android:mimeType="resource/folder" />
```

Comment:
Wasn't this used?

Reply:
This is being used in another intent-filter now.